deepdrivemd/agents/stream/Makefile

```diff
@@ -0,0 +1,6 @@
+project:
```

Comment:
I noticed the other MakeFiles were removed. Can we remove this one too? If not, can we add it to the test directory?

Reply:
The other ones were used for testing during the development. This one is not for testing: I use it to generate embeddings. Perhaps, one can include it into the Makefile in the postproduction_stream directory...

Reply:
Done

Reply:
Thanks! Ideally we want function APIs that we can cleanly call from a jupyter notebook (or elsewhere) which will do things like generate embeddings, other post processing steps, plots, etc.